packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -44,13 +44,19 @@
 			<SplitterPanel
 				class="project-page"
 				v-if="
-					openedWorkflowNodeStore.assetId &&
-					openedWorkflowNodeStore.pageType &&
-					pageType === ProjectAssetTypes.SIMULATION_WORKFLOW
+					(openedWorkflowNodeStore.assetId &&
```

Comment:
Change it to this so it appears only when the workflow is open

`v-if="pageType === ProjectAssetTypes.SIMULATION_WORKFLOW &&
				((openedWorkflowNodeStore.assetId && openedWorkflowNodeStore.pageType) || openedWorkflowNodeStore.node)"`